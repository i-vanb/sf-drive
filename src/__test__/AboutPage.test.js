import {shallow} from "enzyme";
import About from "../About/About";

const wrapper = shallow(<About />)

describe('About', () => {
    it('should be rendered', () => {
        expect(wrapper).toBeDefined()
    });
    it('snapshot matching', () => {
        expect(wrapper).toMatchSnapshot()
    });
});
