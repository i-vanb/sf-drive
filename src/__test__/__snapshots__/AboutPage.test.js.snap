// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About snapshot matching 1`] = `
<Fragment>
  <section
    className="about"
  >
    <div
      className="content-container"
    >
      <img
        alt="drawn team"
        src="about_team.svg"
      />
      <h1>
        О нас
      </h1>
      <p
        className="about__text"
      >
        Это учебный проект, созданный с целью получения боевого опыта в разработке настоящего живого веб-приложения. Этот сервис имитирует работу каршеринга, в котором можно не только арендовать автомобили, но и сдавать их в аренду.
      </p>
    </div>
  </section>
  <section
    className="contacts"
  >
    <div
      className="content-container"
    >
      <h2>
        Контакты
      </h2>
      <div
        className="contacts__data"
      >
        <ContactCard
          data="drive@skillfactory.com"
          name="Электронная почта"
        />
        <ContactCard
          data="+7 912 123-45-67"
          name="Телефон"
        />
      </div>
    </div>
  </section>
  <section
    className="team"
  >
    <div
      className="content-container"
    >
      <h2>
        Команда
      </h2>
      <div
        className="team__cards"
      >
        <ProfileCard
          img="1.png"
          name="Иван Иванов"
          pro="CEO"
        />
        <ProfileCard
          img="2.png"
          name="Алексей Смирнов"
          pro="Frontend-разработчик"
        />
        <ProfileCard
          img="3.png"
          name="Денис Ярцев"
          pro="Backend-разработчик"
        />
        <ProfileCard
          img="4.png"
          name="Николай Морозов"
          pro="Дизайнер"
        />
        <ProfileCard
          img="5.png"
          name="Ирина Деева"
          pro="Копирайтер"
        />
        <ProfileCard
          img="6.png"
          name="Мария Стрелкова"
          pro="SMM"
        />
      </div>
    </div>
  </section>
  <Footer />
</Fragment>
`;
